File: solutions/java/0997.java
class Solution {
  public int findJudge(int N, int[][] trust) {
    int[] count = new int[N];

    for (int[] t : trust) {
      --count[t[0] - 1];
      ++count[t[1] - 1];
    }

    for (int i = 0; i < N; ++i)
      if (count[i] == N - 1)
        return i + 1;

    return -1;
  }
}

File: solutions/python3/0997.py
class Solution:
    def findJudge(self, N: int, trust: List[List[int]]) -> int:
        count = [0] * N

        for t in trust:
            count[t[0] - 1] -= 1
            count[t[1] - 1] += 1

        for i in range(len(count)):
            if count[i] == N - 1:
                return i + 1

        return -1
